augur/utils.py

```diff
@@ -290,7 +290,7 @@ def available_cpu_cores(fallback: int = 1) -> int:
         # naively use all 24, we'd end up with two threads across the 12 cores.
         # This would degrade performance rather than improve it!
         return len(os.sched_getaffinity(0))
-    except:
+    else:
```

Comment:
There's a semantic change here in the edge case, because `os.sched_getaffinity(0)` can fail and raise a Python exception. When it does, Augur will be broken with the change to if/else instead of falling back to `os.cpu_count()` with the original try/except.

I think this change should be reverted and mypy appeased another way.

Reply:
#1251